### PR TITLE
Feature/project track theming

### PR DESCRIPTION
**Pull request by martinyde**
- Added theming for project track.
  - Added Enum for Evaluation values.
  - Opdated status Enum
  - Updated form for project track

### CODE REVIEW

**Review by rimi-itk — CHANGES_REQUESTED** [review]
Looks good. Some minor changes requested.

File: web/modules/custom/ai_screening_project_track/src/Routing/ProjectTrackHtmlRouteProvider.php [hunk 1]
```diff
@@ -20,4 +20,14 @@ protected function getCanonicalRoute(EntityTypeInterface $entity_type): ?Route {
...
 
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditFormRoute(EntityTypeInterface $entity_type) {
+    if ($route = parent::getEditFormRoute($entity_type)) {
+      $route->setOption('_admin_route', FALSE);
+      return $route;
+    }
+  }
```
[the remark this answers is not in the document]
Comment (martinyde) [reply web/modules/custom/ai_screening_project_track/src/Routing/ProjectTrackHtmlRouteProvider.php:32]: Changed

File: web/modules/custom/ai_screening_project_track/src/Form/ProjectTrackForm.php [hunk 2]
```diff
@@ -4,20 +4,117 @@
...
+      '#title' => $this->t('Evaluation: @track_title', ['@track_title' => $this->entity->getTitle()]),
+      '#options' => $this->projectTrackHelper->getEvaluationOptions(),
+      '#default_value' => $this->entity->getProjectTrackEvaluation(),
+    ];
+
+    $form['project_track_status'] = [
+      '#type' => 'select',
+      '#title' => $this->t('Status'),
+      '#options' => $this->projectTrackHelper->getStatusOptions(),
+      '#default_value' => $this->entity->getProjectTrackStatus()->value,
```
[the remark this answers is not in the document]
Comment (martinyde) [reply web/modules/custom/ai_screening_project_track/src/Form/ProjectTrackForm.php:76]: Good as is. One Enum expects int, the other string, that fact seems to surface in the get method.

File: web/modules/custom/ai_screening_project_track/src/Entity/ProjectTrack.php [hunk 3]
```diff
@@ -28,20 +28,29 @@
...
  *     "views_data" = "Drupal\views\EntityViewsData",
- *     "access" = "Drupal\ai_screening_project_track\ProjectTrackAccessControlHandler",
+ *     "access" =
+ *   "Drupal\ai_screening_project_track\ProjectTrackAccessControlHandler",
  *     "form" = {
+ *       "default" = "Drupal\ai_screening_project_track\Form\ProjectTrackForm",
  *       "add" = "Drupal\ai_screening_project_track\Form\ProjectTrackForm",
- *       "edit" = "Drupal\ai_screening_project_track\Form\ProjectTrackForm",
+ *       "edit" =
+ *   "Drupal\ai_screening_project_track\Form\ProjectTrackFormProjectTrackForm",
```
[the remark this answers is not in the document]
Comment (martinyde) [reply web/modules/custom/ai_screening_project_track/src/Entity/ProjectTrack.php]: Changed

**Pull request merged**